contracts/misc/marketplaces/SeaportAdapter.sol

```diff
@@ -24,42 +24,41 @@ contract SeaportAdapter is IMarketplace {
 
     function getAskOrderInfo(bytes memory params)
```

Comment:
getAskOrderInfo and getBidOrderInfo most code logic are the same , could we extract the same part?